### PR TITLE
Pam 220 scopes for oauth apps

### PR DESCRIPTION
**Pull request by pamgoodrich**
Addresses #220 

I changed every topic in the Configure Git section and added a troubleshooting topic. Added a screenshots for GitLab and Bitbucket Cloud. 

Rendered version here:

https://meta.stoplight.io/docs/platform/branches/pam-220-scopes-for-oauth-apps/ZG9jOjQ0MzM2OQ-bitbucket-cloud

### CODE REVIEW

File: 2.-workspaces/configure-git/f-troubleshoot-git.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+- [Bitbucket Cloud](b.bitbucket-cloud.md)
+- [Bitbucket Server](c.bitbucket-server.md)
+- [GitHub Enterprise](d.github-enterprise.md)
+- [GitLab](e.gitlab.md)
+
+## Verify Firewall Settings
+If you are connected behind a firewall, verify that Stoplight's IP addresses are added to your allowlist. 
+
+The fixed IP address is: **35.226.194.249**.
```
Comment (chohmann) [inline 2.-workspaces/configure-git/f-troubleshoot-git.md:25]: Take this or leave this... but should you link to the other page that already lists this? That way if that info ever changes you only have one spot to update?

File: 2.-workspaces/configure-git/f-troubleshoot-git.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## Reinstall Webhooks
```
Comment (chohmann) [inline 2.-workspaces/configure-git/f-troubleshoot-git.md:27]: Maybe it's worth mentioning that they can check the recent deliveries for webhooks in their VCS to get more info on if and why recent webhook deliveries have failed?

image

File: 2.-workspaces/configure-git/f-troubleshoot-git.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+3. Select the **Automation** tab. (This tab is only visible for projects connected to a Git repository.)
+4. Select **Install**.
+
+## Reset Stoplight Access
+
+If connection issues persist, revoke Stoplight authorization in your Git application, and then reauthorize.
+
+## Remove Integration
+
+Consider removing the integration from Stoplight, then confirming you can clone the Git repository to location outside of Stoplight.
```
Comment (chohmann) [inline 2.-workspaces/configure-git/f-troubleshoot-git.md:42]: I think you should mention cloning with `HTTP` and not `SSH`. So maybe

>then confirming you can clone the Git repository using the HTTP/HTTPS URL to a location outside of Stoplight.

That goes with the GitLab tid bit you added of ensuring that cloning with HTTP/HTTPS is enabled.

**Review by chohmann — APPROVED** [review]
This looks great! Just some minor suggestions.

File: 2.-workspaces/configure-git/e.gitlab.md [hunk 4]
```diff
@@ -1,34 +1,40 @@
...
+2. Navigate to **Profile Settings > Edit Profile> Applications** to add a new application via your profile.
+3. Add the following information on the **New Application** form:
+   * **Name**: Enter a Name (e.g `Stoplight Integration`).
+   * **Redirect URI**: Set as `https://stoplight.io/oauth/callback`.
+   * **Expire access token**: Clear this checkbox.
+   * **Scopes**: Select `api`.
+4. Select **Save application**.
 
-**Stoplight Workspace**
+![GitLab Setup](https://stoplight.io/api/v1/projects/cHJqOjI/images/Tr2tmSoaJF8)
```
Comment (ryotrellim) [inline 2.-workspaces/configure-git/e.gitlab.md:18]: Can we update the screenshot to match step 3? Specifically, "Expire access token" should be unchecked.

**Review by ryotrellim — APPROVED** [review]
Just 1 nitpick about the screenshot, otherwise this looks good. Really appreciate the Troubleshooting page!

**Pull request merged**